Automattic/newspack-plugin
# fix(ras-acc): transactional emails helper text and template updates

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1207817176293840/f and https://app.asana.com/0/1207817176293825/1207990881852230/f

Some fixes to email templates and the helper text that appears in wp-admin for the sign in emails:

Sign in emails helper text updated:
![Screenshot 2024-08-08 at 12 28 39](https://github.com/user-attachments/assets/fc30bfa5-5c72-42de-8818-0047db05e6e0)

Template FAQ section update (makes sure the magic link section is present in gmail):
![Screenshot 2024-08-08 at 12 29 33](https://github.com/user-attachments/assets/927eea22-46b9-4dff-b243-8b89a34e76df)


### How to test the changes in this Pull Request:

To test steps 3-5, you will need your test site to send emails to gmail. If not possible, I've updated the login email templates on ras-acc.newspackstaging.com to test there.

1. As admin access the Newspack > Engagement menu, select the Show Advanced Settings button and scroll down to the transactional emails section.
2. Confirm the helper text has been updated as pictured above
3. Select the reset buttons next to both the Magic Link and the OTP email templates
4. Select the edit button next to the Magic link email and in the editor send yourself a test email via the editor side panel.
5. Confirm the FAQ section appears as expected (pictured above)
6. Repeat step 3 and 4 for the OTP email template

### Other information:
